spec/models/split_registry_spec.rb

```diff
@@ -49,39 +49,4 @@
       expect(described_class.new(as_of: 1.day.ago).splits.all).to include(split)
     end
   end
-
-  describe "#experience_sampling_weight" do
-    it "memoizes the env var fetch" do
-      allow(ENV).to receive(:fetch).and_call_original
-
-      subject.experience_sampling_weight
-      subject.experience_sampling_weight
-
-      expect(ENV).to have_received(:fetch).with('EXPERIENCE_SAMPLING_WEIGHT', any_args).exactly(:once)
-    end
-
-    it "returns 1 with no env var" do
-      with_env EXPERIENCE_SAMPLING_WEIGHT: nil do
-        expect(subject.experience_sampling_weight).to eq 1
```

Comment:
Now that this is gone, is there code somewhere that tests the default sampling weight without stubbing?

Reply:
(I get that the tests for the `raise` check are also gone, but I'm just wondering if it's still worth exercising the full codepath at least once in a spec, to avoid surprise breakages in prod.)